src/init.js

```diff
@@ -20,4 +21,5 @@ function initInterceptors() {
 
 export default function init() {
   initInterceptors();
+  initSentry();
```

Comment:
Change to `sentry.init()` for more clarity and a little cleaner.
```suggestion
  sentry.init()
```

Reply:
done!